### PR TITLE
Added organization field in CreateVulnerabilityView

### PR DESCRIPTION
**Pull request by Cheshiriks**
- added organization field in CreateVulnerabilityView

Снимок экрана 2023-07-03 190435

### CODE REVIEW

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt [hunk 1]
```diff
@@ -193,7 +193,7 @@ class VulnerabilityService(
                         vulnerability = vulnerabilityNew,
                         userId = user.requiredId(),
                     )
-                }.toSet()
```
Comment (akuleshov7) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt:196]: Why mutable set? Do you add something to it?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt [hunk 2]
```diff
@@ -275,6 +293,29 @@ val createVulnerabilityView: VFC = VFC {
                                     }
                                 }
 
+                                div {
+                                    className = ClassName("col-12 pl-2 pr-2 mt-3 text-left align-self-center")
+                                    +"Organization"
+
+                                    select {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt:300]: Why not `selectorBuilder`?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt [hunk 3]
```diff
@@ -275,6 +293,29 @@ val createVulnerabilityView: VFC = VFC {
...
+                                        className = ClassName("form-control input-group input-group")
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt:301]:
```suggestion
                                        className = ClassName("form-control input-group custom-select")
```

**Review by sanyavertolet — APPROVED** [review]

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/LnkVulnerabilityUser.kt [hunk 4]
```diff
@@ -9,7 +9,11 @@ import javax.persistence.*
  * @property user in vulnerability with [vulnerabilityId]
  */
 @Entity
+@Suppress("KDOC_NO_CONSTRUCTOR_PROPERTY_WITH_COMMENT")
 class LnkVulnerabilityUser(
+    // Fetching two or more Bags(List) at the same time on an Entity could form a Cartesian Product. Since a Bag doesn't have an order,
```
Comment (nulls) [inline save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/LnkVulnerabilityUser.kt:14]: thanks!

**Review by akuleshov7 — APPROVED** [review]
lgtm

**Pull request merged**